scripts/commands/mobstun.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------------------------------------------------------------
+-- func: mobstun
```

Comment:
Any reason this can't just be `!stun`? and then just check against "anything that isn't an NPC"

Reply:
Renamed this command to `!stun` and adjusted the check to check if not NPC.